python-main.js

```diff
@@ -500,24 +500,40 @@ function web_editor(config) {
         $('.fs-file-list table tbody').empty();
         micropythonFs.ls().forEach(function(filename) {
             var pseudoUniqueId = Math.random().toString(36).substr(2, 9);
-            // Check for main.py to exclude it from the table list
-            if (filename === 'main.py') return;
-            var fileType = (/[.]/.exec(filename)) ? /[^.]+$/.exec(filename) : "";
+            var name = filename;
+            var disabled = "";
+            if (filename === 'main.py') {
+              name = getName() + " (" + filename + ")";
+              disabled = "disabled";
+            };
             $('.fs-file-list table tbody').append(
-                '<tr><td>' + filename + '</td>' +
-                '<td>' + fileType + '</td>' +
+                '<tr><td>' + name + '</td>' +
                 '<td>' + (micropythonFs.size(filename)/1024).toFixed(2) + ' Kb</td>' +
-                '<td><button id="' + pseudoUniqueId + '" class="fs-remove-button">Remove</button></td></tr>'
+                '<td><button id="' + pseudoUniqueId + '2" class="save-button ' + disabled + '" title="Remove" style="width: 4em; height: 2em; float:right;"><i class="fa fa-trash"></i></button>' +
```

Comment:
Can we also change the button background colour to red when hovering it?

Reply:
The text color in blue has very low contrast with the red, do you think white would be better?
![image](https://user-images.githubusercontent.com/29712657/58645434-46ef4980-82fb-11e9-8d00-3c92f0cf75b5.png)

Reply:
Definitely, the css is setup to do this but it does not seem to apply to the icons and I haven't figured out a way to do this yet.